README.md

```diff
@@ -95,6 +96,47 @@ const styles = this.props.location.action === 'POP'
 />
 ```
 
+#### `<AnimatedRoute />`
+Wraps a React Router v4 `<Route />` component in a `<RouteTransition />`. Use this when declaring a custom transition for a single route.
+
+##### Example
+
+```jsx
+<AnimatedRoute
+  path="/show-me"
+  atEnter={{ opacity: 0 }}
+  atLeave={{ opacity: 0 }}
+  atActive={{ opacity: 1 }}
+  component={MyComponent}
+/>
+```
+
+#### `<AnimatedSwitch />`
+Integrates a React Router v4 `<Switch />` component with a `<RouteTransition />`. Use this component when you want a series of Routes to share identical transitions.
+
+```jsx
+// in your root app component:
+
+import { RouteTransition } from 'react-router-transition';
+import { Route, Switch } from 'react-router';
+import { About, Home } from './pages';
+
+<div>
+  <Route render={({location, history, match}) => (
+    <AnimatedSwitch
+      atEnter={{ opacity: 0 }}
+      atLeave={{ opacity: 0 }}
+      atActive={{ opacity: 1 }}
+      key={location.key}
+      location={location}
+    >
+        <Route exact path="/" component={Home} />
+        <Route exact path="/about/" component={About}/>
+    </AnimatedSwitch>
+  )} />
+</div>
```

Comment:
the wrapper will inline the wrapping `Route` component and handle location-passing.